specification/columns/commitmentutilizationtotal.md

```diff
@@ -0,0 +1,44 @@
+# Commitment Utilization Total
+
+Commitment Utilization Total is a numeric value representing the aggregate unit amount of a purchased [*commitment-based discount*](#glossary:commitment-based-discount) for a billing period.  A CommitmentUtilizationTotal value MUST remain a constant value for a specific [*commitment-based discount*](#glossary:commitment-based-discount) within a billing period. The ratio of [*CommitmentUtilizationUsed*](#commitment-utilization-used) over CommitmentUtilizedTotal produces the utilization rate for a given commitment-based discount or aggregate of.
```

Comment:
Should this column represent:
- the total commitment for the time granularity of the line item (e.g., hourly total commitment)
- the total commitment of the entire billing period
- the cumulative total commitment value

Showing it for the granularity of the line item (e.g., hourly) seems to have more detail, but I suppose it depends on what query will be most common.

The AWS column `savings_plan_total_commitment_to_date` gives the cumulative commitment. "The total amortized upfront commitment and recurring commitment to date, for that hour."

Not sure that's necessarily the right definition for FOCUS though.

Reply:
I think showing it by the time granularity of the export (e.g., hour) enables you calculate for any arbitrary time period, what your utilization was

Reply:
Yes, that makes sense and is an oversight on my part.  I think from a cloud commitment perspecitve, we should include the total commitment value at the hourly granularity.

However, for SaaS providers, the commitment may not be evaluated at the hourly granularity (or any granularity) and might be a "use it or lose it" situation.

I think it makes sense to incorporate both models, and I'll amend.